### DISCUSSION TITLE
Native gap support on react-native

### DISCUSSION THREAD

**Discussion by zhigang1992** [nativewind/nativewind#338] [Ideas]
https://github.com/facebook/react-native/pull/34974

https://github.com/reactwg/react-native-releases/discussions/41

This is finally happening!

It would be really cool if we get to use it soon in nativewind. no longer having to use the margin hack.

And we would need to expose a setting somehow for user to configure this if they haven't upgrade to the latest version of react-native.

**Comment by marklawlor** [comment]
I'm going to add this under a flag for the `next` version :100:

**Comment by danstepanov** [comment]
Closing this discussion, if this is still relevant, feel free to open a new one.
